modules/item_transfer/init.lua

```diff
@@ -4,6 +4,15 @@ local module_path = microexpansion.get_module_path("item_transfer")
 
 microexpansion.require_module("network")
 
+
+-- Iron Ingot Ingredient for MineClone2
+iron_ingot_ingredient = nil
```

Comment:
This should probably be prefixed with `microexpansion.` so the variable stays within the microexpansion "namespace".